docs/conf.py

```diff
@@ -94,62 +151,83 @@
     .. role:: raw-html(raw)
         :format: html
 
-    .. note:: This page was generated from `{{ docname }} <https://github.com/azavea/raster-vision/blob/master/docs/{{ docpath }}>`__.
-"""
-#########################
+    .. note:: This page was generated from `{{ docname }} <https://github.com/azavea/raster-vision/blob/master/docs/{{ docpath }}>`__. |ColabLink|_
 
-# -- General configuration ---------------------------------------------------
+.. |ColabLink| image:: https://colab.research.google.com/assets/colab-badge.svg
+.. _ColabLink: https://githubtocolab.com/azavea/raster-vision/blob/master/docs/{{ docpath }}
 
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
 
-# https://read-the-docs.readthedocs.io/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules
-import sys
-from unittest.mock import MagicMock
+.. note::
 
+    If running on Colab:
 
-class Mock(MagicMock):
-    @classmethod
-    def __getattr__(cls, name):
-        return MagicMock()
+    1.  :ref:`Install Raster Vision <install raster vision>`:
 
+        .. code-block:: shell
 
-MOCK_MODULES = ['pyproj', 'h5py', 'osgeo']
-sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
+            pip install rastervision
+    2.  Restart the notebook kernel.
+    3.  Set the ``GDAL_DATA`` environment variable required by ``rasterio``:
+
+        .. code-block:: shell
+
+            export GDAL_DATA=$(pip show rasterio | grep Location | awk '{print $NF"/rasterio/gdal_data/"}')
```

Comment:
To set env vars in a notebook you will need to do something like `%env MY_VAR=MY_VALUE`. Simply running `! export x=y` doesn't work because each invocation of `! ...` runs in its own shell (or something like that).

Reply:
Good call